cognoml/tests/unit_tests/test_utils.py

```diff
@@ -0,0 +1,11 @@
+import os
+import mock
+from cognoml import utils 
+
+@mock.patch('os.mkdir')
+@mock.patch('os.path.exists')
```

Comment:
When we mock a function, instead of actually calling the function, we are just checking whether the function is being called with the correct arguments. In this case, we do not want to create directories, so we are just making sure that when the os.mkdir command is called it is being called with the argument we want.